src/execution/riscv32/simulator-riscv32.cc

```diff
@@ -2590,13 +2585,6 @@ void Simulator::TraceMemRd(int32_t addr, T value, int32_t reg_value) {
                    reg_value, icount_, static_cast<int32_t>(value),
                    static_cast<uint32_t>(value), addr);
           break;
-        case 8:
```

Comment:
If load double-precision float point （fld） is traced，the size would be 8

Reply:
`if (std::is_integral<T>::value) {` this is a int value